observations/pyspark_observation_status_batch.py

```diff
@@ -549,13 +553,39 @@ def observed_data(vts, entype):
 local_path = config.get("OUTPUT_DIR", "observation_status")
 blob_path = config.get("COMMON", "observation_blob_path")
 
+
+fileList = []
+
 for files in os.listdir(local_path):
    if "sl_observation_status.json" in files:
-      cloud_init.upload_to_cloud(blob_Path = blob_path, local_Path = local_path, file_Name = files)
+      fileList.append("sl_observation_status.json")
+
+# Uploading local file to cloud by calling upload_to_cloud fun.
+uploadResponse = cloud_init.upload_to_cloud(filesList = fileList ,folderPathName = "observation_blob_path", local_Path = os.path.join(local_path , str("sl_observation_status.json")))
+successLogger.debug(
+                    "cloud upload response : " + str(uploadResponse)
+                  )
+
+# if file uploading fails exiting the program
+if uploadResponse['success'] == False:
+   sys.exit() 
 
 
 sl_status_spec = {}
+
+#get Druid spec from config
 sl_status_spec = json.loads(config.get("DRUID","observation_status_injestion_spec"))
+
+# updating Druid spec adding type and URI'S
+sl_status_spec["spec"]["ioConfig"]["inputSource"] = uploadResponse['inputSource']
```

Comment:
@adithyadinesh0412 create a function and map cloud path, as we do in all the batch scripts.